pkg/component/autoscaling/vpa/vpa_test.go

```diff
@@ -1405,6 +1420,53 @@ var _ = Describe("VPA", func() {
 				managedResourceName = "vpa"
 			})
 
+			Context("Different ServiceMonitor labels", func() {
+				Context("when used with Garden cluster", func() {
+					BeforeEach(func() {
+						isGardenCluster := true
+						vpa = vpaFor(component.ClusterTypeSeed, isGardenCluster)
+						Expect(vpa.Deploy(ctx)).To(Succeed())
+					})
+
+					It("should label `admission-controller` with `prometheus=garden`", func() {
+						serviceMonitorAdmissionController.ObjectMeta.Name = "garden-vpa-admission-controller"
+						serviceMonitorAdmissionController.ObjectMeta.Labels = map[string]string{
+							"prometheus": "garden",
+						}
+						Expect(managedResource).To(contain(serviceMonitorAdmissionController))
+					})
+
+					It("should label `recommender` with `prometheus=garden`", func() {
+						serviceMonitorRecommender := serviceMonitorRecommenderFor(component.ClusterTypeSeed)
+						serviceMonitorRecommender.ObjectMeta.Name = "garden-vpa-recommender"
+						serviceMonitorRecommender.ObjectMeta.Labels = map[string]string{
+							"prometheus": "garden",
+						}
+						Expect(managedResource).To(contain(serviceMonitorRecommender))
+					})
+
+				})
+
+				Context("when used with non-Garden cluster", func() {
```

Comment:
```suggestion
				Context("when not deployed in a garden cluster", func() {
```

Reply:
Thanks for the suggested wording. You can find the fixed version in my latest updates.